<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerSettings</class>
 <widget class="QWidget" name="TriggerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>306</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="styleSheet">
      <string notr="true">color: white;
	font-size: 14px;
	font-weight: normal;
</string>
     </property>
     <property name="text">
      <string>Trigger Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_under_title">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(74, 100, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>447</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-size: 13px;</string>
           </property>
           <property name="text">
            <string>Trigger mode</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QRadioButton" name="btnAuto">
             <property name="text">
              <string>Auto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="btnNormal">
             <property name="text">
              <string>Normal</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl_source">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 13px;</string>
         </property>
         <property name="text">
          <string>Channel Source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_source">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_internal">
         <property name="styleSheet">
          <string notr="true">font-size: 13px;</string>
         </property>
         <property name="text">
          <string>Internal (Analog)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="adiscope::CustomSwitch" name="intern_en">
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="analog_controls" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_condition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 13px;</string>
            </property>
            <property name="text">
             <string>Condition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_condition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Rising Edge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Falling Edge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="adiscope::PositionSpinButton" name="trigger_level" native="true">
            <property name="name" stdset="0">
             <string>Level</string>
            </property>
            <property name="units" stdset="0">
             <stringlist>
              <string>µVolts=1e-6</string>
              <string>mVolts=1e-3</string>
              <string>Volts=1e0</string>
             </stringlist>
            </property>
           </widget>
          </item>
          <item>
           <widget class="adiscope::PositionSpinButton" name="trigger_hysteresis" native="true">
            <property name="name" stdset="0">
             <string>Hysteresis</string>
            </property>
            <property name="units" stdset="0">
             <stringlist>
              <string>µVolts=1e-6</string>
              <string>mVolts=1e-3</string>
              <string>Volts=1e0</string>
             </stringlist>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_external">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>16</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 13px;</string>
         </property>
         <property name="text">
          <string>External (Digital)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="adiscope::CustomSwitch" name="extern_en">
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="digital_controls" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_extern_condition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 13px;</string>
            </property>
            <property name="text">
             <string>Condition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmb_extern_condition">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Rising Edge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Falling Edge</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Any Edge</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_analog_extern">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 13px;</string>
         </property>
         <property name="text">
          <string>Analog - Digital</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_analog_extern">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Digital OR Analog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Digital AND Analog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Digital XOR Analog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>!Digital OR Analog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>!Digital AND Analog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>!Digital XOR Analog</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder></zorder>
      <zorder>intern_en</zorder>
      <zorder>extern_en</zorder>
      <zorder>analog_controls</zorder>
      <zorder>lbl_external</zorder>
      <zorder>cmb_source</zorder>
      <zorder>cmb_analog_extern</zorder>
      <zorder>lbl_internal</zorder>
      <zorder>lbl_analog_extern</zorder>
      <zorder>digital_controls</zorder>
      <zorder>lbl_source</zorder>
      <zorder>verticalSpacer_3</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>adiscope::CustomSwitch</class>
   <extends>QPushButton</extends>
   <header>customSwitch.hpp</header>
  </customwidget>
  <customwidget>
   <class>adiscope::PositionSpinButton</class>
   <extends>QWidget</extends>
   <header>spinbox_a.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
